src/components/ui/LoadedFilesList.tsx

```diff
@@ -0,0 +1,43 @@
+import { EmailFromList, inputFileName } from 'stores/EmailFormStore'
+import ListedValue from 'components/ui/ListedValue'
+import classnames, { display, flexWrap, gap } from 'classnames/tailwind'
+
+const wrapper = classnames(
+  display('flex'),
+  flexWrap('flex-wrap'),
+  gap('gap-1.5')
+)
+
+export default function ({
+  removeValueFromList,
+  emailList,
+}: {
+  removeValueFromList: (fileName: string) => void
+  emailList: EmailFromList[]
+}) {
+  const files = emailList.map(({ fileName }) => fileName)
+  const uniqueFiles = [...new Set(files)]
+  const filesWithAmount = uniqueFiles.map((file) => ({
```

Comment:
Those 3 lines remap the array of `[{email, fileName, isDifferentDomain}]` into `[{fileName, amount}]`